ksqldb-engine/src/main/java/io/confluent/ksql/engine/InsertValuesExecutor.java

```diff
@@ -100,41 +100,41 @@ public InsertValuesExecutor() {
   public interface RecordProducer {
 
     void sendRecord(
-        ProducerRecord<byte[], byte[]> record,
-        ServiceContext serviceContext,
-        Map<String, Object> producerProps
+            ProducerRecord<byte[], byte[]> record,
```

Comment:
it looks like you have IntelliJ set to use 4 spaces for tabs instead of 2. Please make sure to change that style setting and change this file back :)